packages/edge-login-ui-rn/src/common/locales/en_US.js

```diff
@@ -253,7 +253,31 @@ const strings = {
     'Please enable notifications and background app refresh for account security. Edge will notify you if other devices attempt to access your account and will allow you to accept or reject access.',
   notifications_permissions_android:
     'Please enable notifications for account security. Edge will notify you if other devices attempt to access your account and will allow you to accept or reject access.',
-  change_recovery_same_question: 'Cannot use the same 2 recovery questions'
+  change_recovery_same_question: 'Cannot use the same 2 recovery questions',
+  change_recovery_same_question1:
```

Comment:
Please change all of these names to `change_recovery_question1` and so forth, without the word "same" in there.